app/femr/util/startup/DatabaseSeeder.java

```diff
@@ -1048,4 +1058,16 @@ private void seedAdminUser() {
             userRepository.create(superUser);
         }
     }
+
+    private void seedUserRoles() {
+      // Other roles are in database from early evolutions
+      List<? extends IRole> roles = roleRepository.findAll(Role.class);
+      if (!containRole(roles, "Manager")) {
+        // Manager role doesn't exist, add it
+        Role newRole = new Role();
+        newRole.setId(Roles.MANAGER);
```

Comment:
Any reason for setting the primary key here? I am not sure what the behavior for this would be if a primary key with 7 already exists.